src/js/config/Config.test.js

```diff
@@ -4,7 +4,7 @@ var ConfigDev = {
   rootUrl: '',
   historyServer: '',
   analyticsKey: '39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP',
-  stateRefresh: 2000,
+  stateRefresh: 20000,
```

Comment:
it just helps when running integration tests. It doesn't pollute the heck out of the history that cypress keeps. 

I'm OK with removing this change if you want. I guess it could cause issues in the future, but only when running `npm run testing` which won't run in CI.